pkgcfg/package-configuration/apache/httpd.conf

```diff
@@ -0,0 +1,63 @@
+ServerAdmin             "root@localhost"
+ServerName              "devbox-apache"
+Listen                  "${HTTPD_PORT}"
+PidFile                 "${HTTPD_CONFDIR}/apache.pid"
+
+LoadModule mpm_event_module modules/mod_mpm_event.so
+LoadModule authz_host_module modules/mod_authz_host.so
+LoadModule authz_core_module modules/mod_authz_core.so
+LoadModule auth_basic_module modules/mod_auth_basic.so
+LoadModule mime_module modules/mod_mime.so
+LoadModule headers_module modules/mod_headers.so
+LoadModule unixd_module modules/mod_unixd.so
+LoadModule status_module modules/mod_status.so
+LoadModule proxy_module modules/mod_proxy.so
+LoadModule proxy_fcgi_module modules/mod_proxy_fcgi.so
+LoadModule dir_module modules/mod_dir.so
+LoadModule alias_module modules/mod_alias.so
+
+<IfModule unixd_module>
+    User daemon
+    Group daemon
+</IfModule>
+
+<Directory />
+    AllowOverride none
+    Require all denied
+</Directory>
+
+DocumentRoot  "${PWD}/web"
+<Directory "${PWD}/web">
+    Options Indexes FollowSymLinks
+    AllowOverride None
+    Require all granted
+</Directory>
+
+<Files ".ht*">
+    Require all denied
+</Files>
+ErrorLog "${HTTPD_CONFDIR}/error.log"
+<IfModule headers_module>
+    RequestHeader unset Proxy early
+</IfModule>
+
+<VirtualHost "*:${HTTPD_PORT}">
+    ServerAdmin webmaster@localhost
+    ServerName  php_localhost
+
+    UseCanonicalName    Off
+    DocumentRoot "${PWD}/web"
+
+    <Directory "${PWD}/web">
+        Options All
+        AllowOverride All
+        <IfModule mod_authz_host.c>
+            Require all granted
+        </IfModule>
+    </Directory>
+
+    ## Added for php-fpm
+    # ProxyPassMatch ^/(.*\.php(/.*)?)$ fcgi://{LOCALHOST}:{PHPFPM_PORT}/<?php echo $root ?>/$1
+    DirectoryIndex index.html 
+
+</VirtualHost>
```

Comment:
what resource are you referencing when writing this? Is there a recommended conf on the apache website? It's hard to code review without more knowledge since I'm not an apache expert.

Reply:
@Lagoja this may be a question for you. Where did you get these? we may want to point to the source.